libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/search/FavoritesApi.kt

```diff
@@ -5,8 +5,8 @@ import com.mapbox.androidauto.car.feedback.core.CarFeedbackSearchOptionsProvider
 import com.mapbox.androidauto.car.placeslistonmap.PlacesListOnMapProvider
 import com.mapbox.bindgen.Expected
 import com.mapbox.bindgen.ExpectedFactory
-import com.mapbox.search.AsyncOperationTask
 import com.mapbox.search.CompletionCallback
+import com.mapbox.search.common.AsyncOperationTask
```

Comment:
Flagging as this causes a incompatibility between 1.0.0-beta.35 and 1.0.0-beta.36